dlrover/python/master/node/job_manager.py

```diff
@@ -190,3 +199,17 @@ def sync_node_training_port(self, node_id, port) -> SyncNodeTrainingPorts:
         return self._training_node_configure.sync_node_training_port(
             node_id, port
         )
+
+    def update_node_required_info(self, min_required, max_required):
```

Comment:
Add docstring to explain the `min_required` and `max_required`.

Print a warning log if `min_required > max_required`.

Reply:
done